mne/viz/utils.py

```diff
@@ -2883,6 +2883,7 @@ def _plot_masked_image(ax, data, times, mask=None, yvals=None,
             im = ax.imshow(
                 np.ma.masked_where(~mask, data), cmap=cmap, **im_args)
         else:
+            ax.imshow(data, cmap=cmap, **im_args)  # see #6481
             im = ax.imshow(data, cmap=cmap, **im_args)
```

Comment:
Drawing the same thing twice is pretty weird. Did you try doing a `ax.figure.draw_idle()` here instead? That would feel cleaner than plotting the same thing twice. But if it doesn't work, I can live with this.

Reply:
I tried

`ax.get_figure().canvas.draw_idle()`

but it does not fix the pb.